MAUI/TreeMap/selection-and-highlights.md

```diff
@@ -0,0 +1,548 @@
+---
+layout: post
+title: Selection and highlights in .NET MAUI TreeMap | Syncfusion
+description: Learn about the functionality of selection and highlights within the Syncfusion .NET MAUI TreeMap(STreeMap) control.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Selection in TreeMap control
+
+The Treemap control allows interactive selection of items by clicking or tapping. `Tooltip`, if enabled, displays additional information upon clicking or tapping, enhancing user understanding of data points. When `SelectionMode` is `None`, interaction is disabled for treemap items.
+
+## Selection mode
+
+Selection mode in the Treemap control determines the interactivity level, enabling users to choose specific items by clicking or tapping. The default value of `SelectionMode` is `SelectionMode.Single`.
+
+### Single selection
+With `Single` selection mode, you can choose one item at a time within the Treemap control, ideal for isolating and examining individual data points thoroughly.
```

Comment:
In the Single selection mode, users can choose one item at a time within the Treemap control.